app/assets/locales/messages_ccodk_default.txt

```diff
@@ -304,6 +304,8 @@ demo.mode.feature.unavailable=Sorry, this app uses a feature that is not availab
 
 select.search.label=Search:
 select.search.status=${0} of ${1} results for your search: "${2}"
+select.callout.search.status=${0} of ${1} results for callout result

```

Comment:
Yeah, I agree. See the second screenshot in the PR desc. for where this comes up. Open to better suggestions; we can also address this after someone does a UX pass...